### PR TITLE
alerts : alerts created

### PR DESCRIPTION
**Pull request by aamrapaliwandhre**

### CODE REVIEW

File: Components/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.alert{
+    display: flex;
+    align-items: center;
+    width: 95%;
+    max-width: 1000px;
+    height: 55px;
+    margin: 25px 10px 25px 10px;
+    padding-left: 20px;
```
Comment (aman11s) [inline Components/alert/alert.css]: I would suggest you not use an absolute value like px instead use rem. This will help you in responsiveness.

**Comment by aman11s** [comment]
I had given some suggestions. Apart from it, the rest looks good.

**Review by aman11s — APPROVED** [review]

File: Components/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    border-radius: 5px;
+    font-size: 1.2rem;
+}
+.alert-error{
+    background-color: rgb(247, 59, 59);
```
Comment (SanjayJatti) [inline Components/alert/alert.css:13]: You can add these colors to root by creating variables. So that you can reuse it.

**Review by SanjayJatti — APPROVED** [review]
Apart from suggestion, code is clean. You can go ahead.

**Pull request merged**